### PR TITLE
6/2 post-update hotfixes and changes

### PR DESCRIPTION
**Pull request by ghost**
- gar/garb/s-gar inhands have been pruned slightly to be more efficient and space-saving because most of them are only a pixel different
- butterfly knife inhand removed, uses switchblade inhand because they're nearly identical and the butterfly is never used
- ion rifle inhand fixed
- telebaton changes, it no longer fits in backpacks when it's 'on' and it also has a slightly shorter weaken time because it basically works as a 100% disarm shove right now
- large clothing placement overhaul, moving around a lot of the clothing, especially in sec lockers, and making a lot of colored jumpsuits probability -- you never know what you might get in a mixed locker! (they'll be a lot less bloated now)
- fixed sec jumpsuit rolldown finally (it's actually an issue with armbands i think but this will solve 99% of problems, sorry security jumpsuit rollers, the sprites will remain in the uniform.dm)
- sec officers spawn with a corporate poster in their lockers, warden gets 3, out-propaganda the enemy!
- military jacket no longer has the SUSPICIOUSLY SLAVIC FLAG looking patch on the breast, armpatches look more like NT logos, comes in the sec officer alt wardrobe with cargo pants on a prob
- STERILE MASKS CONCEAL YOUR IDENTITY WITHOUT ID LIKE BANDANAS, YOU TOO CAN BE IN A JAPANESE YAKUZA GANG!
- leather jackets have a small chance to spawn in black wardrobes and casual lockers, but are still poppin-collar #rare and a decent reward for good botanists who have the time to make them

### CODE REVIEW

**Comment by ghost** [comment]
Also can someone figure out why the game can't compile this? Something to do with the else clauses in wardrobe.dm, do I have to completely change how the lockers work in order to use else clauses or something? (like how the job lockers decide between backpack/satchels)

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -136,11 +143,14 @@
 	New()
 		..()
 		sleep(2)
-		new /obj/item/clothing/suit/labcoat/coat/security(src)
-		new /obj/item/clothing/suit/armor/vest(src)
+
 		if(prob(35))
 			new /obj/item/clothing/suit/armor/vest/jacket(src)
+		else
+			new /obj/item/clothing/suit/labcoat/coat/security(src)
```
Comment (Atskadan) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: dont make this a probability
muh winter coats
youll make me cry

**Comment by GunHog** [comment]
Add the click delay to the telebaton! You can spam the noise and people's chat windows. More annoying than rapid fire clown honking.

File: code/game/objects/structures/crates_lockers/closets/wardrobe.dm [hunk 2]
```diff
@@ -84,18 +94,20 @@
 	new /obj/item/clothing/shoes/sneakers/black(src)
 	new /obj/item/clothing/shoes/sneakers/black(src)
 	new /obj/item/clothing/shoes/sneakers/black(src)
-	new /obj/item/clothing/shoes/sneakers/black(src)
-	new /obj/item/clothing/shoes/sneakers/black(src)
-	new /obj/item/clothing/shoes/sneakers/black(src)
-	if(prob(75))
+	if(prob(30))
+		new /obj/item/clothing/suit/labcoat/coat/jacket/leather(src()
```
Comment (GunHog) [inline code/game/objects/structures/crates_lockers/closets/wardrobe.dm]: Might want to tack a ")" unto this so it compiles.

Comment (RemieRichards) [inline code/game/objects/structures/crates_lockers/closets/wardrobe.dm]: Actually no @GunHog it's not

`new /obj/item/clothing/suit/labcoat/coat/jacket/leather(src())`
as you suggest

it's

`new /obj/item/clothing/suit/labcoat/coat/jacket/leather(src)`

Comment (Atskadan) [inline code/game/objects/structures/crates_lockers/closets/wardrobe.dm]: remie you poop get in irc

**Comment by ghost** [comment]
Ah! Thank you for finding it gun_hog and remie!

**Comment by ghost** [comment]
@Atskadan on the winter coats -- but muh bloat. This way you can get either, and 65% of the time in the sec lockers it'll be winter coats. The winter coats are in the wardrobe locker on a 100% anyway, I added them.

Edit: I changed it up a bit. Winter coats are now back of the sec lockers 100% like all the other job lockers, but the military jackets are wardrobe only -- however there is more of a chance for them to spawn.

Does anyone -- besides desucake -- Oppose me moving the 2nd wardrobe locker in the sec checkpoint now that it has armor in it? I can think of a much better place on metastation for it.

**Comment by ghost** [comment]
- TELEBATON HAS A COOLDOWN. YOU CAN CHAINSTUN STILL, BUT JUST BARELY, BASICALLY ANYTHING CAN INTERRUPT IT.
- CAPTAIN'S GLOVES ARE INSULATED (WHY WASN'T THIS A THING)
- COMPILING FIX
- WINTER COATS IN OFFICER LOCKERS 100%
- MORE CHANCE FOR MILITARY JACKETS AND CAMO PANTS IN WARDROBE FOR TACTICOOL
- CAP CARAPACE, LABCOATS, AND CE RIG CAN HOLD TELEBATON IN THEIR SUIT STORAGE

**Comment by Atskadan** [comment]
yay

**Comment by ghost** [comment]
I'll ask again, does anyone oppose me removing the wardrobe from sec checkpoint/fortifying the sec checkpoint with r-walls now that it has armor in it?

If people are sour about not being able to steal jackboots as easily i can always add some to maint.

**Comment by ghost** [comment]
ss 2014-06-02 at 04 53 17
ss 2014-06-02 at 04 56 00

Done. Those are legit posters in the sec checkpoint now.

Edit: I moved that air vent back under the lower wardrobe, not sure why I moved it. I also added two pairs of jackboots, hidden in maint.

**Comment by ghost** [comment]
- Mutli-belt var-edited to fix the typo.
- Two pairs of jackboots in maint.
- Air vent moved back.
- No smoking sign in chemistry.

**Comment by ghost** [comment]
Added https://github.com/NTStation/NTstation13/pull/668 to this

**Comment by ghost** [comment]
- Sec belt holds the telebaton now
- Posters are all over the map in maint, 5-6 maybe total because they had to be taken down because of a bug with the new 'legit' posters
- Commented out AI nightvision, but it should still be a cheap malf module!

**Comment by Neerti** [comment]
Why the stealthnerf?

**Comment by ghost** [comment]
What stealth nerf? It's an oversight by gun hog and wasn't supposed to be in here in the first place. He stealth re-added it a DAY after he removed it, which was dumb because the only person who opted for it to stay in was you.

Let me break it down for you because while I already explained this to gun hog on irc it needs to be said again apparently.

It's wickedly overpowered and makes camera lights irrelevant. Silicons are NOT supposed to have night vision, all of their weaknesses are focused around their vision, and that includes hiding from them in the dark, much like constructs. Silicons are completely and utterly robust otherwise. Overloading their vision and hiding in the dark is the trade off counter for being robust, unstunnable, and being completely unaffected by atmospherics. Gun Hog said it was a 'minor hassle' in the post, dealing with lighting, but I know what he really meant, which is that robust players know to smash lights and hide from the AI.

Another reason? The AI can overload lighting in an area and still be able to see, which is literally removing the tradeoff from overloading the lights. What's to stop an antag AI from just overloading all the lights in the station again after flooding plasma? Oh right, fucking nothing. Traitors can't run into darkness to avoid being tracked by the AI as well, which is similarly awful.

Stop adding this shit without thinking it through. Night vision should at best be a malf AI module, but this is stupid as fuck.

**Comment by ghost** [comment]
Before someone mentions "b-but muh gamma tweaking for experienced veteran AI players!"

That's an exploit and you shouldn't be doing it anyway. Hell, if you have to do that to circumvent darkness, you're just spoiling the game for yourself in the end. I haven't heard of a single person except a handful of tryhard dedicated AI players who do it anyway. It's an exploit that needs to be fixed, and it won't be fixed by you adding ezmode vision to AIs.

**Comment by ghost** [comment]
Added a shorter variant of the over_eye hair style which is basically long fringe but over eye for all you wannabe japanese schoolgirls out there

**Comment by GunHog** [comment]
It is _not_ an oversight, and as I recall, you were the only one truly opposed to it. I re-added it thinking such that more were for it than were against, doing so as quickly as possible as the bulk of my votes were already in the positive before the removal anyway.

I would appreciate a poll on it rather than you simply gutting it. I apologize if I offended you by not giving you notification of it, or if anyone else opposing it changed their opinion of the feature based on that.

**Comment by ghost** [comment]
Yeah no. I gave you my reasons for taking it out and if that alone doesn't make you say 'wow that was stupid' I'm pretty sure there's no convincing you.

Again, this is a balance issue, thanks for taking the time to read my reasons for taking it out instead of ignoring them because you want the AI to be overpowered. Oh, nevermind.

Unless you refute them with a substantial opinion or fact based argument, I'm not taking this out of my PR.

**Comment by ghost** [comment]
It only applies to help/grab/disarm intents which are the 'knockdown' intents, so there's cooldown on knocking people over which is basically the same effect as the disarm 'push'. It's harder to chainstun with the baton on help in other words, because it's already an unlimited 100% chance to 3-tick weaken. Harm intent works like any other 10 brute melee weapon, no knockdown.

Give it a try if you'd like to see the difference physically, but it's noticably less spammable and works more like a head's self defense weapon if someone rushes them with a stun prod or flash or something. Straight up melee against a head that knows what they're doing is basically impossible now, which is the point behind my telebaton (bay has since removed the knockdown from their telebaton because they had issues balancing it) so maybe heads of staff will be played more now that they have an anti-tide weapon and can shit on retarded intruders until security arrives.

Also fine, if there's really some great debate to be had about a pretty fucking obvious (at least to me) balance issue then I'll make a new PR for it.

**Comment by ghost** [comment]
Not touching this anymore, merge this and then merge JStheguy's poster PR.

**Comment by ghost** [comment]
lemme cheq

**Comment by ghost** [comment]
sorry i can't hear you over the 5 billion important things i updated that were more serious problems and not an unused switchblade resprite that happens to be bloating up the inhands

snark aside, yeah, i found the issue and am working on fixing it.

**Comment by ghost** [comment]
OK SO
- SWITCHBLADE REPLACED WITH BUTTERFLY KNIFE IN BASIC CONSTRUCTION, SWITCHBLADE IS MUCH MORE COMPLEX / HAS GOLD FEATURES AND LOOKS MORE LIKE A GANGSTER GIMMICK WEAPON FOR ADMINS TO SPAWN THAN AN IMPROVISED WEAPON LIKE THE BUTTERFLY. BUTTERFLY TWEAKED. BUTTERFLY IS NEW IMPROVISED BLADE, STATS ARE UNCHANGED.
- BATS ARE 1 BRUTE LESS, THEY'RE BASICALLY STUNLESS STUN BATONS NOW, W/E
- METAL BAT ADDED TO HACKED AUTOLATHE
- !!BAT INHANDS!!

THAT IS ALL, AND YES IT'S TESTED, IF SOMETHING REGARDING THIS BUGS IN THE FUTURE PLS KILL ME

**Pull request merged**